pyproject.toml

```diff
@@ -29,6 +29,7 @@ dependencies = [
     "pandas>=1.1.3;python_version<'3.10'",
     "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
     "pandas>=1.5.0;python_version>='3.11'",
+    "scikit-learn>=0.24,<2;python_version<'3.10'",
```

Comment:
can we add more expanded support? I think python 3.10 can support scikit-learn 1.1.0 so we don't need to make the minimum for 3.10 1.1.3

Reply:
I see, I took it from SDMetrics based on last week bug. Maybe we should update it there also:
https://github.com/sdv-dev/SDMetrics/blob/eee51755bc105288b726673f698bf8f36980b3de/pyproject.toml#L31

Reply:
Can you add that to this PR, and I can also approve it? I think I did not see that comment before merging.

Reply:
After discussing, we found that RDT has actually pinned scikit-learn between 1.1.3 and 2 for python 3.10 for a while now, so we can just leave this